### DISCUSSION TITLE
[Webhooks] Add testing button

### DISCUSSION THREAD

**Discussion by jon4hz** [btcpayserver/btcpayserver#2442] [General]
I think it would be nice to have a button to manually trigger the webhook to test if the configured endpoint is working, instead of waiting until an actual event happens.

**Comment by bolatovumar** [comment]
This sounds like a reasonable request. Stripe allows testing webhooks. We could provide something similar. See here: https://stripe.com/docs/webhooks/test

cc @pavlenex

**Reply by bolatovumar** [reply]
@jon4hz made a draft PR: https://github.com/btcpayserver/btcpayserver/pull/2474. It's still in draft mode for now but it allows you to test webhooks similar to how Stripe allows you to do it.

See example test webhook call:

![Screen Shot 2021-04-18 at 5 56 33 PM](https://user-images.githubusercontent.com/1934678/115168362-78986780-a06f-11eb-8d24-aee818d5c488.png)
